harness/determined/searcher/_search_method.py

```diff
@@ -240,15 +344,60 @@ def on_trial_closed(
         self, searcher_state: SearcherState, request_id: uuid.UUID
     ) -> List[Operation]:
         """
-        Informs the searcher that a trial has been closed as a result of a Close
-        operation.
+        Informs the searcher that a trial has been closed as a result of a :class:`~Close`
+
+        Example:
+
+        .. code:: python
+
+            def on_trial_closed(
+                self, searcher_state: SearcherState, request_id: uuid.UUID
+            ) -> List[Operation]:
+                MAX_NUM_TRIALS = 200  # Best defined in __init__
+                if searcher_state.trials_created < MAX_NUM_TRIALS:
+                    hparams = {
+                        # ...
+                    }
+                    return [
+                        searcher.Create(
+                            request_id=uuid.uuid4(),
+                            hparams=hparams,
+                            checkpoint=None,
+                        )
+                    ]
+                if searcher_state.trials_closed >= MAX_NUM_TRIALS:
+                    return [searcher.Shutdown(failure=searcher_state.experiment_failed)]
```

Comment:
Similar question to the above here: is it actually possible for `searcher_state.experment_failed` to be `True` when reaching this code? Didn't think so.

Reply:
Added